docs/samples/browser-plugin-meetings/app.js

```diff
@@ -434,12 +434,18 @@ meetingsListElm.onclick = (e) => {
     document.getElementById('btn-join').disabled = true;
     document.getElementById('btn-join-media').disabled = true;
   }
-  else if (meeting && meeting.passwordStatus === 'UNKNOWN') {
+  else if (meeting && (meeting.passwordStatus === 'UNKNOWN' && meeting.destinationType!="")) {
```

Comment:
```suggestion
  else if (meeting && (meeting.passwordStatus === 'UNKNOWN' && meeting.destinationType==="Person ID")) {
```
Rather than having an empty array can we be more specific. This will make sure these changes only affect the 1:1 meeting and not other flows.

Reply:
@Shreyas281299 ,That was I had keep an empty array because of the response of the meeting object.When I was given the Person ID or Email as destination type and click on create meeting button.The meeting object given the response as json and we stored it as the meeting variables.From that we taken Password Status and Destination type.For Person_ID or Email the destination type is "",so I had keep an empty array.I had tried to keep the Destination type as the option we selected but it won't Working to create a meeting and giving the meeting object where destination type is empty array.

Reply:
Got it. In that case can we use [this](https://github.com/webex/webex-js-sdk/blob/4798f7a5d1a4b0123fac40eea05acb815ce1ddc0/docs/samples/browser-plugin-meetings/app.js#L282) element and get the exact value of the destination and use that here.

Reply:
Hi @Shreyas281299 Regarding the taking as attributes(values="PERSON ID") or innertext of html tag(Option section in Manage Meeting) we cannot able to create meeting and taken default PERSON ID as destination output instead of email.Need to discuss with the @mkesavan13 for his input.